pkgs/development/libraries/concord/default.nix

```diff
@@ -12,16 +8,13 @@ stdenv.mkDerivation rec {
     owner = "Cogmasters";
     repo = "concord";
     rev = "v${version}";
-    hash = "sha256-+nuVyqA/V/J6DWAeIs9Pv90ry3px1gJsF460qvfSEH8="; # Fill this in based on the fetch error
+    hash =
+      "sha256-+nuVyqA/V/J6DWAeIs9Pv90ry3px1gJsF460qvfSEH8=";
   };
 
-  buildInputs = [
-    curl
-  ];
+  buildInputs = [ curl ];
```

Comment:
Please keep them on seperate lines, makes future additions/removals/changes easier to diff